packages/dd-trace/src/log_channels.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const dc = require('diagnostics_channel')
+
+const Level = {
+  Debug: 'debug',
+  Info: 'info',
+  Warn: 'warn',
+  Error: 'error'
+}
+
+const _defaultLevel = Level.Debug
+
+class LogChannel {
+  constructor (name, logLevel) {
+    this.channel = dc.channel(`dd-trace:log:${name}`)
```

Comment:
```suggestion
    this.channel = dc.channel(`datadog:log:${name}`)
```

Reply:
Looks like this change was reverted?

Reply:
I had not seen that comment, sorry. It is restored again.